tools/selection_tools.cpp

```diff
@@ -606,6 +606,39 @@ void SelectSubsetEdgesByDirection(
 	                       maxDeviationAngle, selectFlipped);
 }
 
+Edge* SelectEdgeByMinPerpendicularDistance(
```

Comment:
As commented somewhere, this is not accurate - if the projection of the given coordinate on an edge lies outside of that edge, the distance to the closest endpoint is considered.